lib/assert.js

```diff
@@ -150,6 +152,14 @@ x.ifError = (err, msg) => {
 	test(!err, create(err, 'Error', '!==', msg, x.ifError));
 };
 
+x.jsxEqual = (val, expected, msg) => {
+	test(deepEqual(val, expected), create(val, expected, '===', msg, x.jsxEqual));
+};
+
+x.notJsxEqual = (val, expected, msg) => {
+	test(!deepEqual(val, expected), create(val, expected, '!==', msg, x.notJsxEqual));
+};
```

Comment:
I don't think there's any now.

I do have one idea for these, but not sure if that can be implemented to be stable enough. We (or community) could write a babel plugin for react/preact/inferno, which detects `t.jsxEqual` and convert:

```jsx
t.jsxEqual(<This/>, <That/>);
```

into:

```jsx
t.jsxEqual(renderer.create(<This/>).toJSON(), renderer.create(<That/>).toJSON());
```

to save the user from typing that manually or creating shortcuts (as mentioned in #1175).